pom.xml

```diff
@@ -1878,20 +1878,44 @@ org.hibernate.search.spi.SearchIntegratorBuilder#buildSearchIntegrator()
                 <!-- Asciidoctor and Karaf won't work on JDK9 yet  -->
                 <jdk>1.8</jdk>
             </activation>
+            <properties>
+                <!-- Surefire versions are a minefield of bugs, so depending on context we need to choose a different one -->
+                <maven.plugin.surefire.version>2.20.1</maven.plugin.surefire.version>
+                <maven.plugin.failsafe.version>2.20.1</maven.plugin.failsafe.version>
```

Comment:
Would you mind if I simply made this the default, and removed the overrides for jdk8 and jdk9? I'm afraid we'll forget to update both the next time we upgrade...

Reply:
ok. The reason I didn't set a default is to catch mistakes, I never trust its "inheritance rules" ;)